senza/manaus/constants.py

```diff
@@ -0,0 +1,11 @@
+ELB_REGION_HOSTED_ZONE = {"us-east-1": "Z35SXDOTRQ7X7K",
```

Comment:
Didn't we agreed in [another PR](https://github.com/zalando-stups/senza/pull/237) to not have a list of AWS zones?